requirements.txt

```diff
@@ -1,170 +1,165 @@
-pytz==2022.4 ; python_version >= "3.9" and python_version < "3.10"
-pyxb==1.2.5 ; python_version >= "3.9" and python_version < "3.10"
-pyyaml==6.0 ; python_version >= "3.9" and python_version < "3.10"
-razorpay==1.3.0 ; python_version >= "3.9" and python_version < "3.10"
-redis==4.3.4 ; python_version >= "3.9" and python_version < "3.10"
-requests-oauthlib==1.3.1 ; python_version >= "3.9" and python_version < "3.10"
-requests==2.28.1 ; python_version >= "3.9" and python_version < "3.10"
-rsa==4.9 ; python_version >= "3.9" and python_version < "3.10"
-rx==1.6.1 ; python_version >= "3.9" and python_version < "3.10"
-s3transfer==0.6.0 ; python_version >= "3.9" and python_version < "3.10"
-sendgrid==6.9.7 ; python_version >= "3.9" and python_version < "3.10"
-sentry-sdk==1.9.10 ; python_version >= "3.9" and python_version < "3.10"
-setuptools==65.5.0 ; python_version >= "3.9" and python_version < "3.10"
-six==1.16.0 ; python_version >= "3.9" and python_version < "3.10"
-sniffio==1.3.0 ; python_version >= "3.9" and python_version < "3.10"
-soupsieve==2.3.2.post1 ; python_version >= "3.9" and python_version < "3.10"
-sqlparse==0.4.3 ; python_version >= "3.9" and python_version < "3.10"
-starkbank-ecdsa==2.1.0 ; python_version >= "3.9" and python_version < "3.10"
-stripe==3.5.0 ; python_version >= "3.9" and python_version < "3.10"
-sympy==1.11.1 ; python_version >= "3.9" and python_version < "3.10"
-text-unidecode==1.3 ; python_version >= "3.9" and python_version < "3.10"
-threadloop==1.0.2 ; python_version >= "3.9" and python_version < "3.10"
-thrift==0.16.0 ; python_version >= "3.9" and python_version < "3.10"
-tinycss2==1.1.1 ; python_version >= "3.9" and python_version < "3.10"
-tornado==6.2 ; python_version >= "3.9" and python_version < "3.10"
-types-redis==4.3.21.2 ; python_version >= "3.9" and python_version < "3.10"
-typing-extensions==4.4.0 ; python_version >= "3.9" and python_version < "3.10"
-tzdata==2022.5 ; python_version >= "3.9" and python_version < "3.10"
-urllib3==1.26.12 ; python_version >= "3.9" and python_version < "3.10"
-uvicorn[standard]==0.18.3 ; python_version >= "3.9" and python_version < "3.10"
-uvloop==0.17.0 ; sys_platform != "win32" and sys_platform != "cygwin" and platform_python_implementation != "PyPy" and python_version >= "3.9" and python_version < "3.10"
-vine==5.0.0 ; python_version >= "3.9" and python_version < "3.10"
-watchfiles==0.17.0 ; python_version >= "3.9" and python_version < "3.10"
-wcwidth==0.2.5 ; python_version >= "3.9" and python_version < "3.10"
-weasyprint==56.1 ; python_version >= "3.9" and python_version < "3.10"
-webencodings==0.5.1 ; python_version >= "3.9" and python_version < "3.10"
-websockets==10.3 ; python_version >= "3.9" and python_version < "3.10"
-wrapt==1.14.1 ; python_version >= "3.9" and python_version < "3.10"
-zipp==3.9.0 ; python_version >= "3.9" and python_version < "3.10"
-zopfli==0.2.1 ; python_version >= "3.9" and python_version < "3.10"
+adyen==4.0.0
```

Comment:
Can you use Poetry≥1.2.0? Looks like it was generating via Poetry <1.2.0 thus the diffs are way too big

Reply:
true, I updated Poetry and generated files one more time.